### ISSUE TITLE
Should schema be versioned in alignment with the context?

### ISSUE THREAD

**Issue by kayaelle** [1EdTech/openbadges-specification#536]
The context for Open Badges is at https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.2.json

The id for the AchievementCredential is: https://purl.imsglobal.org/spec/ob/v3p0/schema/json/ob_v3p0_achievementcredential_schema.json

**Comment by xaviaracil** [comment]
Yes, they are. We do the follow the [1EdTech Versioning Framework](https://www.imsglobal.org/spec/versioning/v1) for this type of artifacts
